src/openlcb/MultiConfiguredPC.hxx

```diff
@@ -138,10 +138,20 @@ public:
         ConfigUpdateService::instance()->register_update_listener(this);
         producedEvents_ = new EventId[size * 2];
         std::allocator<debouncer_type> alloc;
+// C++20 standard removed the construct and destruct methods from
+// std::allocator which are now available in std::allocator_traits requiring
+// passing in the allocator to these methods.
+#if __cplusplus >= 202002L
```

Comment:
We don't need to make this code conditionalized. std::allocator_traits exists since c++11, and we are using c++14 currently (never intend to go back to below c++11).